server/linter/analyzer/analyzer_entities.go

```diff
@@ -40,18 +50,94 @@ func (l Linter) Validate() error {
 	}
 
 	for _, p := range l.Plugins {
-		if p.Name == "" {
-			return fmt.Errorf("plugin name cannot be empty")
+		plugin, ok := findPlugin(p.Slug, metadata.Plugins)
```

Comment:
This is now validating the incoming configuration to contain the available plugins and rules registered in the backend